fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -592,7 +593,7 @@ public void testPatchBinaryDescription() throws Exception {
         patch.addHeader("Content-Type", "application/sparql-update");
         final BasicHttpEntity e = new BasicHttpEntity();
         e.setContent(new ByteArrayInputStream(
-                ("INSERT { <" + location +
+                ("INSERT { <" + serverAddress + pid + "/x" +
```

Comment:
No. Obviously, the test passes, so that can't be the case.

The simple sense is that descriptions contain triples about the described thing, so only triples with the described thing as their subject are legal.